language/generated-samples/v1/language_sentiment_text.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "analyze_sentiment")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-language
+
+import sys
+
+# [START language_sentiment_text]
+
+from google.cloud import language_v1
+from google.cloud.language_v1 import enums
```

Comment:
is it possible to not import this separately, but use it as `language.enums` in the code?